tableaudocumentapi/xml_open.py

```diff
@@ -0,0 +1,28 @@
+try:
+    from distutils2.version import NormalizedVersion as Version
+except ImportError:
+    from distutils.version import LooseVersion as Version
+
+import xml.etree.ElementTree as ET
+import zipfile
+
+from tableaudocumentapi import xfile
+
+MIN_SUPPORTED_VERSION = Version("9.0")
+
+
+class VersionNotSupportedException(Exception):
+    pass
+
+
+def xml_open(filename):
+    # Determine if this is a twb or twbx and get the xml root
```

Comment:
nit: tds and tdsx too... maybe just say "if this is an xfile or not" ;) -- we really need a word for those 'x' files we have